.github/workflows/refresh-lockfiles.yml

```diff
@@ -108,13 +108,24 @@ jobs:
           rm -r artifacts
         
       - name: Create Pull Request
-        uses: peter-evans/create-pull-request@052fc72b4198ba9fbc81b818c6e1859f747d49a8
+        id: cpr
+        uses: peter-evans/create-pull-request@v3
```

Comment:
We can pin to the latest stable release... unless there's a specific reason we need to use the previously pinned commit

Reply:
This is considered unsafe practice with third party actions (i.e. those not made by GitHub), as a bad actor could re-label a new malicious commit as `v3`.

Reply:
Well, at the very least we can bump it to a later specific commit

Reply:
also we should start thinking about using dependabot

Reply:
Dependabot is ace but doesn't support conda files

Reply:
https://github.com/dependabot/dependabot-core/issues/2227

Reply:
@jamesp I was thinking more of using dependabot to automate (or tell us) when the `create-pull-request` GHA itself can be updated to a later release (rather than using it to resolve/check for later releases of conda packages in the lock files)

It should be able to do that, right?

Reply:
ah yes, good idea. it looks like dependabot scans any gha in our yaml files. https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-the-dependency-graph#dependencies-included
So long as the GHA code uses one of the supported ecosystems, I guess

Reply:
As a starter, see #4568